client/blocks/reader-share/index.jsx

```diff
@@ -70,73 +84,75 @@ function buildQuerystringForPost( post ) {
 	return qs.stringify( args );
 }
 
-const ReaderShare = React.createClass( {
+function canShareToWordPress() {
+	return !! user.get().primarySiteSlug;
+}
+
+class ReaderShare extends React.Component {
 
-	propTypes: {
+	static propTypes = {
 		iconSize: React.PropTypes.number
-	},
+	}
 
-	getInitialState() {
-		return { showingMenu: false };
-	},
+	static defaultProps = {
+		position: 'bottom',
```

Comment:
Yep. We need `bottom` in both places it's used (bottom of post card and devdocs) so it seemed like a more sensible default.